javers-core/src/test/groovy/org/javers/core/JaversRepositoryE2ETest.groovy

```diff
@@ -85,17 +85,14 @@ class JaversRepositoryE2ETest extends Specification {
     def 'should persist current LocalDateTime and Instant in CommitMetadata' () {
         given:
         def now = ZonedDateTime.now()
-        println now
         setNow(now)
 
         when:
         javers.commit('author', new SnapshotEntity(id: 1))
         def snapshot = javers.getLatestSnapshot(1, SnapshotEntity).get()
 
         then:
-        println 'commitDate: ' + snapshot.commitMetadata.commitDate
-        println 'commitDateInstant: ' + snapshot.commitMetadata.commitDateInstant
-        snapshot.commitMetadata.commitDate == now.toLocalDateTime()
+        ChronoUnit.MILLIS.between(snapshot.commitMetadata.commitDate, now.toLocalDateTime()) == 0
```

Comment:
It turned out that Spock test cannot be easily overridden in a subclass, therefore I have to introduce this change here and not in MySqlIntegrationTest.